interfaces/cython/cantera/test/test_thermo.py

```diff
@@ -1643,37 +1643,37 @@ def test_slicing_ndim(self):
 
     def test_extra(self):
         with self.assertRaises(ValueError):
-            states = ct.SolutionArray(self.gas, extra=['creation_rates'])
-        states = ct.SolutionArray(self.gas, shape=(0,0), extra=("prop1"))
-        self.assertEqual(states.prop1.shape,(0,0))
+            states = ct.SolutionArray(self.gas, extra = ['creation_rates'])
+        states = ct.SolutionArray(self.gas, shape = (0, 0), extra = ("prop1"))
+        self.assertEqual(states.prop1.shape, (0, 0))
 
         # test indexed operations on extra arrays
-        states = ct.SolutionArray(self.gas, 7, extra={'n': range(7)})
+        states = ct.SolutionArray(self.gas, 7, extra = {'prop': range(7)})
```

Comment:
I have other thoughts about these changes, but for Python, `=` for keyword arguments should not have a space around them.